compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  ner:
+    image: linhdpostdata/hismetag
+    ports:
+      - "8080:8080"
+  api:
+    image: linhdpostdata/poetrylab-api
+    environment:
+    - NER_URI=http://ner:8080/api/rest/process/text
```

Comment:
Not sure if using 8080 would be a problem because of another service using that port